code/go/0chain.net/smartcontract/dbs/event/edb_transaction.go

```diff
@@ -0,0 +1,15 @@
+package event
+
+import (
+	"0chain.net/smartcontract/dbs"
+	"gorm.io/gorm"
+)
+
+type edbTransaction struct {
```

Comment:
Let's use edbTx. We might use Txn for 0chain transaction and Tx for all other transaction as a convention later.